flowpipe/node.py

```diff
@@ -434,12 +448,23 @@ def _initialize(self, func, outputs, metadata):
             if arg_spec.defaults is not None:
                 defaults = dict(zip(arg_spec.args[-len(arg_spec.defaults):],
                                     arg_spec.defaults))
+            forbidden_inputs = []
             for input_ in arg_spec.args:
+                if input_ in FunctionNode.RESERVED_INPUT_NAMES:
```

Comment:
Does it make sense replace `FunctionNode.RESERVED_INPUT_NAMES` with `self.RESERVED_INPUT_NAMES`? That way, a subclass of FunctionNode just has to extend the variable and immediately inherits the functionality.